src/Features/Core/Portable/Structure/BlockTypes.cs

```diff
@@ -4,25 +4,40 @@ namespace Microsoft.CodeAnalysis.Structure
 {
     internal static class BlockTypes
     {
-        public const string AccessorBlock = "AccessorBlock";
-        public const string AnonymousMethodBlock = "AnonymousMethodBlock";
-        public const string Case = "Case";
-        public const string Class = "Class";
-        public const string Conditional = "Conditional";
-        public const string Constructor = "Constructor";
-        public const string Destructor = "Destructor";
-        public const string Interface = "Interface";
-        public const string Lock = "Context";
-        public const string Loop = "Loop";
-        public const string Method = "Method";
-        public const string Namespace = "Namespace";
-        public const string Nonstructural = "Nonstructural";
-        public const string Operator = "Operator";
-        public const string PropertyBlock = "PropertyBlock";
-        public const string Standalone = "Standalone";
-        public const string Struct = "Struct";
-        public const string Structural = "Structural";
-        public const string TryCatchFinally = "TryCatchFinally";
-        public const string Unknown = "Unknown";
+        // Basic types.
+        public const string Structural = nameof(Structural);
+        public const string Nonstructural = nameof(Nonstructural);
```

Comment:
I made it this way to match the editor side.  They actually started as NonStructural but moved ot Nonstructural.